specs/0108-community-health-dashboard.md

```diff
@@ -33,13 +34,27 @@ Active users within a selected time frame are defined like this:
 
 Note that only users included in the all users counts can be active users. If a project contact person rejects a suggestion or submits a translation, it doesn't count as an active reviewer or contributor of the team.
 
-## Unreviewed suggestion lifespan
+## Average age of currently unreviewed suggestions
 
-A line chart showing the average age of the unreviewed suggestions in a particular month for a period of last 12 months.
+A line chart showing the average age of suggestions still unreviewed in a particular month, over the last 12 months.
 
-Data is available for each day, but aggregated by month in the chart.
+Data is available for each day, but aggregated by month in the chart. A tooltip showing the exact data at a given month appears when hovering over a chart.
+
+**Text for infobox**
+
+    How old are on average the suggestions currently unreviewed.
+
+## Time to review suggestions
+
+A line chart displaying two separate data points over the last 12 months:
+* *Monthly reviews*: the average age of suggestions reviewed during a specific month.
+* *Average 12 months*: the average age of suggestions reviewed in the 12 months before the month currently examined.
+
+In both cases, data is determined by looking at translations rejected or approved, ignoring self-approved or self-rejected translations.
+
+**Text for infobox**
 
-A tooltip showing the exact data at a given month appears when hovering over a chart.
+    The average number of days a contributor can expect to wait for their translation suggestions to be reviewed by managers and translators.
```

Comment:
This is a good title for the infobox. We'll also need to explain each data point, same as we do in the activity charts. Probably with the same bullet points as above.

I also suggest we use different styling in the spec - the blockquote results in a horizontal scroll.

Reply:
Remove the blockquote formatting for now. Do we need to literally repeat the two bullet points in the infobox here, just add a note explaining it, or something else?

Reply:
A note would be sufficient.